### PR TITLE
Adds a new inventory to vendors, now they dispense special items once emagged!

### PR DESCRIPTION
**Pull request by Carbonhell**
Yay!
Also does the following:
1) Adds two new bottles, an Hooch one and an Hippie's Delight one, for Booze-o-mat.
2)A new stimpack with 10 units of meth, available through a vendor
Should be that

### CODE REVIEW

**Comment by Carbonhell** [comment]
Whoops!It also adds a toy claymore that's the exact copy of the claymore, without the can block thingy and without the op force, and fixes a little thing with bandolier

**Comment by Carbonhell** [comment]
New commit fixes #628

**Comment by danhibiki** [comment]
Sure why not

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -269,7 +275,8 @@
 /obj/machinery/vending/emag_act(user as mob)
 	if(!emagged)
 		emagged  = 1
-		user << "You short out the product lock on [src]."
+		user << "You short out the product lock on [src] and unlock some items."
```
Comment (SplergeBucket) [inline code/game/machinery/vending.dm:278]: This really necessary? I think the "shorted out product lock" is good enough.

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -610,6 +629,7 @@
 					/obj/item/weapon/reagent_containers/food/drinks/soda_cans/cola = 8, /obj/item/weapon/reagent_containers/food/drinks/soda_cans/sodawater = 15,
 					/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 30,/obj/item/weapon/reagent_containers/food/drinks/ice = 9)
 	contraband = list(/obj/item/weapon/reagent_containers/food/drinks/tea = 10)
+	syndicate = list(/obj/item/weapon/reagent_containers/food/drinks/bottle/hooch = 10, /obj/item/weapon/reagent_containers/food/drinks/bottle/hippie = 10)
```
Comment (SplergeBucket) [inline code/game/machinery/vending.dm:632]: und what do these two do?

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -650,6 +671,7 @@
 	vend_delay = 34
 	products = list(/obj/item/weapon/reagent_containers/food/drinks/coffee = 25,/obj/item/weapon/reagent_containers/food/drinks/tea = 25,/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 25)
 	contraband = list(/obj/item/weapon/reagent_containers/food/drinks/ice = 10)
+	syndicate = list(/obj/item/weapon/reagent_containers/hypospray/medipen/methpack = 1)
```
Comment (SplergeBucket) [inline code/game/machinery/vending.dm]: i fucking want lcass chem reverted already aaaaaaahhh

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Carbonhell) [inline code/game/machinery/vending.dm:278]: It's just a message so people know what happens once you emag the machines, if you want me to remove it it's fine i guess

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (Carbonhell) [inline code/game/machinery/vending.dm:632]: both are alcohol which bartender can easily make,hooch is a strong alcohol made with ethanol,welding fuel and sugar, while hippie delight is a normal alcohol like pan galactic gargle blaster, all i did was creating bottles with them in it

**Comment by SplergeBucket** [comment]
with meth gone i'd say all gud

**Comment by Carbonhell** [comment]
Meth is gone,MERGE

**Pull request merged**